### PR TITLE
improvements to properties.schema

### PR DESCRIPTION
**Pull request by moloko**
remove elearning jargon 'hotgraphic' from component description, improve titles and help text

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -106,7 +105,7 @@
       "title": "Cycle through item pagination",
       "inputType": "Checkbox",
       "validators": [],
-      "help": "If enabled, the items in the open popup will be allowed to cycle through continuously"
+      "help": "If enabled, the popup navigation buttons will cycle continuously through the popup items (i.e. clicking next on the last item will take them to the first one)."
```
Comment (brian-learningpool) [inline properties.schema]: Can we replace 'them' with 'the user' here?

File: properties.schema [hunk 2]
```diff
@@ -198,17 +197,17 @@
...
-                "help": "Title text for this image",
+                "help": "Title text for the popup image",
                 "translatable": true
               },
               "_classes": {
                 "type": "string",
                 "required": false,
                 "default": "",
-                "title": "Classes",
+                "title": "CSS Classes",
```
Comment (brian-learningpool) [inline properties.schema]: I'm not sure about this.  None of the other `_classes` properties are actually labelled as "CSS Classes", including the component level `_classes` in this same file.

File: properties.schema (hunk 1 above)
Comment (moloko) [reply properties.schema]: yep my bad I was actually trying to write this in such a way that it didn't reference the learner/user but forgot to adjust this bit. How about 'clicking next on the last item will cause the first item to be shown'?

File: properties.schema (hunk 2 above)
Comment (moloko) [reply properties.schema]: OK fair enough

File: properties.schema (hunk 1 above)
Comment (brian-learningpool) [reply properties.schema]: Perfect.

Comment (oliverfoster) [reply properties.schema]: Does this need doing still?

Comment (moloko) [reply properties.schema]: it does, good spot

File: properties.schema [hunk 3]
```diff
@@ -172,15 +171,15 @@
                 "default": "",
                 "inputType": "Asset:image",
                 "validators": ["required"],
-                "help": "Image displayed in the popup"
+                "help": "Image to be displayed in a popup when this item is selected."
```
Comment (tomgreenfield) [inline properties.schema]: Did you mean to add a full stop 🇬🇧 / period 🇺🇸 here? 😉

Comment (moloko) [reply properties.schema]: I did, yes. And yes, I know there's no consistency as to whether `help` text should include a full stop or not. Happy to make it consistent if we can agree on which style it should be.

Comment (tomgreenfield) [reply properties.schema]: Just looking at the rest of the file, it looks like full stops have only been bothered with if there's more than one sentence. Happy to go with that for now and not set a steadfast rule.

**Pull request merged**